addons/stock/static/src/widgets/generate_serial.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<templates>
+    <t t-name="stock.GenerateSerials">
+        <button class="btn btn-link" t-on-click="openDialog">Generate Serials/Lots</button>
+    </t>
+</templates>
```

Comment:
I would put both this and the other template you are creating in the existing `lots_dialog.xml` file since they are related (they are the 2 buttons that will open up the template that is in that file). You can put it at the top of the file so they're obvious (due to their short nature).

I would also name them consistently since they're for the same thing (both buttons can generate and import serials/lots), so:

keep `stock.ImportLots`
change other one to `stock.GenerateLots`

ideally the existing template (`stock.generate_serial_dialog`) would also match the naming scheme, but it's already inconsistent (since it's already snake case and it says serial, but its template is in lots_dialog). So let's stick with `Lots` since the model is `stock.lot`

Reply:
Usually the naming convention for component templates is `module.ComponentName`, though